templates/supplier_app/ap-taxpayers.html

```diff
@@ -55,6 +55,11 @@ <h1>
                         <a class='btn btn-info' href="{% url 'supplier-details' taxpayer_id=taxpayer.id %}">
                             {% trans "More details" %}
                         </a>
+                        {% if taxpayer.new_comment_from_supplier%}
+                        <a data-toggle="tooltip" data-placement="right" title="{% trans 'New unread commments' %}">
+                            <i class='far fa-comment-dots' style='font-size:24px'></i>
```

Comment:
nit: add aria-hidden="true" for accesibility reasons
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-hidden_attribute